index.ts

```diff
@@ -0,0 +1,41 @@
+import { multiUrlGenerator } from "./multiUrlGenerator";
+
+const clientUI = require("readline").createInterface({
+  input: process.stdin,
+  output: process.stdout,
+});
+
+class Url {
+  $home: string;
+  $name: string;
+  constructor($home: string, $name: string) {
+    this.$home = $home;
+    this.$name = $name;
+  }
+  createObject(): any {
+    return {
+      $home: this.$home,
+      $name: this.$name,
+    };
+  }
+}
+
+function arrayOfObjectsFactory(website: string[], path: string[]) {
+  let voidArray = [];
+  for (let i = 0; i < website.length; i++) {
+    let obj: {} = new Url(website[i], path[i]).createObject();
+    voidArray.push(obj);
+  }
+  return voidArray;
+}
+
+clientUI.question("type <website>, <routes> => ", function (userInput: string) {
```

Comment:
I believe this is ok as it is, since this is just placeholder code that will not represent the final usage of the library. Ideally, we'll want to provide a .json file for this script to read the URLs from. Let's leave this as it is and move on to the next task.